.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -60,13 +93,46 @@ jobs:
     displayName: "Cleanup ACI"
     pool:
       vmImage: ubuntu-20.04
-    dependsOn: ${{ parameters.used_by }}
+    dependsOn:
+      - deploy_aci
```

Comment:
There should be a transitive dependency with "used_by" so is this needed?

Reply:
This is to access output variables of `deploy_aci`  by `$[ dependencies.deploy_aci.outputs['deploy_aci.resource_group'] ]` for example